### DISCUSSION TITLE
Blockchain token

### DISCUSSION THREAD

**Discussion by IliaHashemi** [WeAreMahsaAmini/Core#221] [Ideas]
Creating a blockchain token (crypto) with the aim of supporting protesters in Iran.
Project goals:
- The world's first blockchain token related to supporting a revolution
- Fundraising for protesters in Iran
- Introducing Iran's revolution and protests to all the people of the world and cryptocurrency activists
- Spiritual value
- Financial value for investors and supporters
- Using a decentralized system for financial assistance

Road map:
- Select token name and blockchain network
- Determining smart contract programming
- Contract programming
- Determination of Tokenomics (IEO, IDO, ICO Airdrop stages)
- Implementation of the main smart contract for the project and initial supply contract and airdrop
- Implementation and development of contracts on web3
- Development and completion of whitepaper and road map
- Implementation of the initial launch landing page through Web3 interact
- Verify Trust Wallet and BSC Scan
- Implementing the initial supply
- Implementation of smart contracts to start the IDO phase and CoinMarketCap verification
- Start three rounds of IDO through sales from landing page and DEX

**Comment by mrnobud** [comment]
Love the concept, and would love to help (I have experience working in blockchain companies in US) 

A few points: 
- Creating a new blockchain is hard, time consuming and requires a lot of dev work. We probably need to build an ERC-20 or BEP, etc.
- If the only benefit you are seeking is financial benefits and fund raising we can achieve the same result by creating an NFT project 
- If we don't have utility for the token and only build it for collection or financial assistance, it can be manipulated and/or perceived as pure scam for money laundry. 
- Getting the token on any major exchange / DEX would be very hard, if not impossible, would require market makers, LPs, etc. Getting investment would be hard as we have to navigate through financial laws and  sanctions. 


I suggest we create a [DAO](https://github.com/WeAreMahsaAmini/Core/discussions/225)
It is still on the blockchain, and we will benefit from anything that a blockchain has to offer, but we don't need to build a blockchain project from the ground up or be worry about getting listed or scams.

**Reply by mkianpour** [reply]
Another great benefit of having this Token would be ability to vote when some decisions need to be made

p.s - I heard that there is already a scam token by the name of Mahsa Amini :(

**Comment by ghost** [comment]
Please check the following link:
https://github.com/WeAreMahsaAmini/Core/discussions/114#discussioncomment-3764485
